astronomer/providers/amazon/aws/example_dags/example_aws_nuke.py

```diff
@@ -125,6 +131,15 @@ def check_dag_status(**kwargs: Any) -> None:
         f"aws emr-containers list-virtual-clusters --state RUNNING --region {AWS_DEFAULT_REGION} | jq -r '.virtualClusters[].id' | xargs -I % aws emr-containers delete-virtual-cluster --id % --region {AWS_DEFAULT_REGION}; ",
     )
 
+    terminate_regression_clusters = BashOperator(
+        task_id="terminate_regression_clusters",
```

Comment:
let's mention here that these are from Astro SDK
e.g. terminate_astro_sdk_regression_clusters or terminate_dag_authoring_regression_clusters

Reply:
terminate_dag_authoring_regression_clusters would be better